schemas/config/3.0.schema.json

```diff
@@ -1,2600 +1,2869 @@
+									"type": "string",
+									"examples": [
+										"Cache-Control,Content-Language,Content-Type,Expires,Last-Modified,Pragma"
+									]
+								},
+								"max_age": {
+									"description": "Controls the value of 'Access-Control-Max-Age' (in seconds).",
+									"type": "integer",
+									"examples": [
+										600
+									],
+									"default": 0
+								}
+							}
+						},
+						"request": {
+							"description": "Headers to add to every request passed to PHP.",
+							"$ref": "#/definitions/HashmapString"
+						},
+						"response": {
+							"description": "Headers added to every response.",
+							"$ref": "#/definitions/HashmapString"
+						}
+					}
+				},
+				"static": {
+					"description": "Configuratin options for serving static files.",
+					"type": "object",
+					"additionalProperties": false,
+					"properties": {
+						"dir": {
+							"description": "Path to the directory with static assets. Defaults to the current working directory. Empty/undefined and `.` are equal and are both treated as current directory.",
+							"type": "string",
+							"examples": [
+								".",
+								"/var/www/html"
+							]
+						},
+						"forbid": {
+							"description": "File extensions of files that must not be served. Empty/undefined disallows no files. If files are specified in both `forbid` and `allow`, they will be disallowed. Defaults to an empty array, disallowing no files.",
+							"type": "array",
+							"items": {
+								"type": "string",
+								"minLength": 1,
+								"examples": [
+									".php",
+									".htaccess",
+									".sh"
+								]
+							}
+						},
+						"allow": {
+							"description": "File extensions of files that may be served. Empty/undefined allows all files, except files specified in `forbid`.",
+							"type": "array",
+							"items": {
+								"type": "string",
+								"minLength": 1,
+								"examples": [
+									".jpg",
+									".png",
+									".css",
+									".js"
+								]
+							}
+						},
+						"calculate_etag": {
+							"description": "Whether to enable ETag computation for static files.",
+							"type": "boolean",
+							"default": false
+						},
+						"weak": {
+							"description": "Whether to use a weak generator (/W), which uses only the filename to generate a CRC32 sum for et ETag. Disable to use the file contents.",
+							"type": "boolean",
+							"default": false
+						},
+						"response": {
+							"description": "Custom HTTP headers to add to responses to requests for static files.",
+							"$ref": "#/definitions/HashmapString"
+						},
+						"request": {
+							"description": "Custom HTTP headers to add to requests for static files.",
+							"$ref": "#/definitions/HashmapString"
+						}
+					}
+				},
+				"pool": {
+					"description": "Settings for workers in the HTTP pool.",
+					"$ref": "#/definitions/WorkersPool"
+				},
+				"ssl": {
+					"title": "SSL/TLS (HTTPS) Configuration",
```

Comment:
Added validation logic that requires *either* `acme` or `key` and `cert`, but not both at the same time.